server/src/main/java/org/elasticsearch/search/builder/SearchSourceBuilder.java

```diff
@@ -895,6 +910,21 @@ public SearchSourceBuilder stats(List<String> statsGroups) {
         return stats;
     }
 
+    /**
+     * Add a search time mapping.
+     */
+    public SearchSourceBuilder extraMapping(Map<String, Object> extraMapping) {
```

Comment:
runtimeMappings?

Reply:
Sure. I think I prefer "searchTime" but we've been calling them runtime for a while now.